docs/input.md

```diff
@@ -108,6 +108,115 @@ which results in:
 
 ![Ask circle diameter](img/circle_radius.png){: loading=lazy }
 
+
+### Check for mouse presses
+
+Spark features multiple ways to check for user mouse presses:
+
+| Name                                  | Type                  | Description                                                            |
+|---------------------------------------|-----------------------|------------------------------------------------------------------------|
+| [mouse_is_pressed](#mouse_is_pressed) | Variable              | A boolean variable that is True when a mouse button is pressed         |
+| [mouse_down()](#mouse_down)           | User defined function | A definable function that activates when a mousebutton is pressed down |
```

Comment:
Consistency: Use `mouse button` instead of `mousebutton`

Reply:
Resolved in latest commit